code/modules/research/bepis.dm

```diff
@@ -2,11 +2,11 @@
 
 //economics defined values, subject to change should anything be too high or low in practice.
 
-#define MACHINE_OPERATION 100000
-#define MACHINE_OVERLOAD 500000
-#define MAJOR_THRESHOLD 3000
-#define MINOR_THRESHOLD 2000
-#define STANDARD_DEVIATION 1000
+#define MACHINE_OPERATION 400*CARGO_CRATE_VALUE
+#define MACHINE_OVERLOAD 2000*CARGO_CRATE_VALUE
+#define MAJOR_THRESHOLD 12*CARGO_CRATE_VALUE
+#define MINOR_THRESHOLD 8*CARGO_CRATE_VALUE
+#define STANDARD_DEVIATION 4*CARGO_CRATE_VALUE
```

Comment:
To keep the pricing consistent with how crates were balanced, it should probably be 6x for major, 4x for minor, and 2x for the standard deviation.

Reply:
SO i was right the first time!